### ISSUE TITLE
INamedType.ToDisplayString omits delegate's parameters

### ISSUE THREAD

**Issue by andriipatsula** [dotnet/arcade#11232]
**Version Used**: 4.3.0

**Steps to Reproduce**:

SymbolDisplayFormat options:
```C#
var format = new SymbolDisplayFormat(
        globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.Omitted,
        typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces,
        delegateStyle: SymbolDisplayDelegateStyle.NameAndSignature,
        miscellaneousOptions: SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
```

Code example:
```C#
namespace A
{
    public delegate bool SyntaxReceiverCreator(int a, bool b);
}
```

**Expected Behavior**:
as `delegateStyle: SymbolDisplayDelegateStyle.NameAndSignature` is active, I expect `INamedType.ToDisplayString(format)` produces output:
```
bool SyntaxReceiverCreator(int a, bool b)
```

**Actual Behavior**:
`INamedType.ToDisplayString(format)` produces output (without parameters):

```
bool A.SyntaxReceiverCreator()
```

From parameter description:
> **SymbolDisplayDelegateStyle.NameAndSignature** - Shows the name and the signature of the delegate (e.g. "void SomeDelegate(int x)").

Issue in Roslyn backlog: https://github.com/dotnet/roslyn/issues/64632

**Comment by andriipatsula** [comment]
@RikkiGibson, I created an issue https://github.com/dotnet/roslyn/issues/64632 and this one is a mirror in dotnet/arcade.
Besides that, I'd like to discuss one more think. In example above the `INamedType.ToDisplayString` produces output `bool A.SyntaxReceiverCreator()` where `A` is a namespace and `SyntaxReceiverCreator` is a delegate name. In case SyntaxReceiverCreator is inside a class like
```C#
namespace A
{
  class Foo
  {
    public delegate bool SyntaxReceiverCreator(int a, bool b);
  }
}
```

the output 'd be `A.Foo.SyntaxReceiverCreator`. In SymbolDisplayParameter I specified `typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces`, but I expected it's related to type definitions, to a delegate.

/cc @tkapin
